gtk/src/callbacks.c

```diff
@@ -577,6 +579,70 @@ check_name_template(signal_user_data_t *ud, const char *str)
     return FALSE;
 }
 
+int match(const char *string, const char *pattern)
```

Comment:
This should be a static function to prevent name space pollution.  "match" is a very generic name and likely to collide with something someday.